sway/commands/permit.c

```diff
@@ -40,9 +41,14 @@ static enum secure_feature get_features(int argc, char **argv,
 
 static struct feature_policy *get_policy(const char *name) {
 	struct feature_policy *policy = NULL;
+	char *program = NULL;
+
+	if (!(program = resolve_path(name))) {
+		sway_abort("Unable to allocate security policy");
```

Comment:
abort is probably not the correct action here, use sway_assert and return a policy with no permissions.